mgmt/rest/client/metric.go

```diff
@@ -112,6 +114,10 @@ func (c *Client) GetMetric(ns string, ver int) *GetMetricResult {
 	case rbody.MetricReturnedType:
 		mc := resp.Body.(*rbody.MetricReturned)
 		r.Metric = mc.Metric
+	case rbody.MetricsReturnedType:
+		mc := resp.Body.(*rbody.MetricsReturned)
+		r1 := convertMetrics(mc)
```

Comment:
Agreed but this implies having the return r1 becomes return r too because it can not be removed (so as to be factorized with the final return. Is this ok with you ?